addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3344,11 +3387,11 @@ var SnippetsMenu = Widget.extend({
         const range = selection && selection.rangeCount && selection.getRangeAt(0);
         if (!range ||
             !$(range.commonAncestorContainer).parents('#wrapwrap, .iframe-editor-wrapper .o_editable').length ||
-            $(selection.anchorNode).parent('[data-oe-model]:not([data-oe-type="html"]):not([data-oe-field="arch"])').length ||
-            $(selection.focusNode).parent('[data-oe-model]:not([data-oe-type="html"]):not([data-oe-field="arch"])').length ||
+            $(selection.anchorNode).parent('[data-oe-model]:not([data-oe-type="html"]):not([data-oe-field="arch"]):not([data-oe-translation-id])').length ||
```

Comment:
**BEFORE MERGE**

@detrouxdev Things like this make me this those are stuff which are needed outside of the context of this PR, am I wrong? Should not it be in its dedicated PR, maybe even targeting stable?

Reply:
Yes, I am removing this change and opened a PR here: https://github.com/odoo/odoo/pull/93806